src/patternfly/components/SkipToContent/skip-to-content.hbs

```diff
@@ -1,6 +1,8 @@
-<a class="pf-c-skip-to-content pf-c-button pf-m-primary{{#if skip-to-content--modifier}} {{skip-to-content--modifier}}{{/if}}"
-  {{#if skip-to-content--attribute}}
-    {{{skip-to-content--attribute}}}
-  {{/if}}>
-  {{> @partial-block}}
-</a>
+<div class="pf-c-skip-to-content">
+  <a class="pf-c-button pf-m-primary{{#if skip-to-content--modifier}} {{skip-to-content--modifier}}{{/if}}"
+    {{#if skip-to-content--attribute}}
+      {{{skip-to-content--attribute}}}
+    {{/if}}>
+    {{> @partial-block}}
+  </a>
+</div>
```

Comment:
This looks good as it is, but there is one small cleanup we could make in this PR, or we can open another issue and handle that separately, just lemme know.

The `--modifier` and `--attribute` attrs should ideally go on the component level thing in the hbs file, so here we would move them to the new `div.skip-to-content` element. That means we'll need to use something other than `skip-to-content--attribute='href="#target"'` to give the link an `href`.

We also have a `button-link` partial that generates a link using the button component. One way could be to do it this way below, creating a `skip-to-content--href` param to pass the URL in as:

```suggestion
<div class="pf-c-skip-to-content{{#if skip-to-content--modifier}} {{skip-to-content--modifier}}{{/if}}"
  {{#if skip-to-content--attribute}}
    {{{skip-to-content--attribute}}}
  {{/if}}>
  {{#> button-link button-link--attribute=(concat 'href="' skip-to-content--href '"') button-link--modifier="pf-m-primary"}}
    {{> @partial-block}}
  {{/button-link}}
</div>
```

Then anywhere we're using `{{#> skip-to-content [attr param with href]}}` just use `{{#> skip-to-content skip-to-content--href="#target"}}`

Looks like we would want to update these spots:

https://github.com/patternfly/patternfly/blob/e362c95792e234592911881cb3c818177c023fd5/src/patternfly/demos/NotificationDrawer/notification-drawer-demo.hbs#L2-L4

https://github.com/patternfly/patternfly/blob/e362c95792e234592911881cb3c818177c023fd5/src/patternfly/demos/Page/page-template-skip-to-content.hbs#L1-L3

https://github.com/patternfly/patternfly/blob/e362c95792e234592911881cb3c818177c023fd5/src/patternfly/components/SkipToContent/examples/SkipToContent.md#L10-L12

https://github.com/patternfly/patternfly/blob/e362c95792e234592911881cb3c818177c023fd5/src/patternfly/demos/Page/page-demo-expandable-nav.hbs#L2-L4